masonry/src/widget/widget_state.rs

```diff
@@ -61,6 +67,11 @@ pub struct WidgetState {
     // TODO - Document
     pub(crate) is_portal: bool,
 
+    pub(crate) request_compose: bool,
+    // TODO - Handle matrix transforms
```

Comment:
Matrices are tricky with text, due to optical size of text (which we don't yet handle). I think changing the scale of a child can only correctly happen in `layout`

Reply:
I'm leaving that debate for once we do a complete layout refactor.